dot-net-sdk/EppoClient.cs

```diff
@@ -220,13 +238,22 @@ public static EppoClient Init(EppoClientConfig eppoClientConfig)
                 modelCache,
                 banditFlagCache);
 
-            var expConfigRequester = new ConfigurationRequester(eppoHttpClient, configurationStore);
-            s_client?._fetchExperimentsTask.Dispose();
+            var configRequester = new ConfigurationRequester(eppoHttpClient, configurationStore);
+            s_client?._fetchExperimentsTask?.Dispose();
 
-            var fetchExperimentsTask = new FetchExperimentsTask(expConfigRequester, Constants.TIME_INTERVAL_IN_MILLIS,
-                Constants.JITTER_INTERVAL_IN_MILLIS);
-            fetchExperimentsTask.Run();
-            s_client = new EppoClient(expConfigRequester, eppoClientConfig, fetchExperimentsTask);
+            FetchExperimentsTask? fetchExperimentsTask = null;
+            if (appDetails.Deployment == SDKDeploymentMode.SERVER)
```

Comment:
Configuration is **only** loaded when `RefreshConfiguration` is called in client mode.